src/main/java/greynekos/greybook/logic/commands/UnmarkCommand.java

```diff
@@ -0,0 +1,119 @@
+package greynekos.greybook.logic.commands;
+
+import static greynekos.greybook.logic.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import static greynekos.greybook.logic.Messages.MESSAGE_MISSING_STUDENTID;
+import static greynekos.greybook.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Optional;
+
+import greynekos.greybook.commons.core.index.Index;
+import greynekos.greybook.logic.Messages;
+import greynekos.greybook.logic.commands.exceptions.CommandException;
+import greynekos.greybook.logic.parser.ArgumentParseResult;
+import greynekos.greybook.logic.parser.GreyBookParser;
+import greynekos.greybook.logic.parser.ParserUtil;
+import greynekos.greybook.logic.parser.commandoption.OptionalSinglePreambleOption;
+import greynekos.greybook.model.Model;
+import greynekos.greybook.model.person.AttendanceStatus;
+import greynekos.greybook.model.person.Person;
+import greynekos.greybook.model.person.StudentID;
+
+/**
+ * The UnmarkCommand clears a club member's attendance status.
+ */
+public class UnmarkCommand extends Command {
+
+    public static final String COMMAND_WORD = "unmark";
+    public static final String UNMARK_ALL_KEYWORD = "all";
+
+    public static final String MESSAGE_UNMARK_PERSON_SUCCESS = "Cleared %1$s's attendance status";
+    public static final String MESSAGE_UNMARK_ALL_SUCCESS = "All attendance status have been cleared.";
```

Comment:
Would it be helpful to track how many statuses have been unmarked?

```suggestion
    public static final String MESSAGE_UNMARK_ALL_SUCCESS = "Cleared attendance status for %d member(s).";
```

Reply:
Sure, sounds good. I will implement those changes